config/locales/en.yml

```diff
@@ -326,3 +326,11 @@ en:
               min_grade:          "must be greater than 0"
             user:
               blank:              "can't be blank'"
+        assignment:
+          attributes:
+            user:
+              blank:                 "User can't be null"
```

Comment:
i18n comments do not include the name of the model, "can't be blank" will be filled